docs/concepts/tokens/fungible-tokens/multi-purpose-tokens.md

```diff
@@ -10,48 +11,78 @@ status: not_enabled
 
 _(Requires the [MPTokensV1 amendment][] {% not-enabled /%})_
 
-Multi-purpose tokens (MPTs) are a more compact and flexible type of fungible token.
+Multi-purpose tokens (MPTs) are a more compact and flexible type of [fungible token](/docs/concepts/tokens/fungible-tokens) on the XRP Ledger.
```

Comment:
This type of link is fine, but when the link is _this_ proximate I think it's more appropriate to use relative linking, e.g.:

```suggestion
Multi-purpose tokens (MPTs) are a more compact and flexible type of [fungible token](./index.md) on the XRP Ledger.
```